src/content/en/updates/2019/01/rtcquictransport-api.md

```diff
@@ -0,0 +1,246 @@
+is to move towards lower level APIs, and experimenting early with this is
+valuable.
+
+### Why QUIC?
+The QUIC protocol is desirable for real time communications. It is built on top
+of UDP, has built in encryption, congestion control and is multiplexed without
+head of line blocking. The RTCQuicTransport gives very similar abilities as the
+RTCDataChannel API, but uses QUIC rather than SCTP as its transport protocol.
+Because the RTCQuicTransport is a standalone API, it doesn’t have the overhead
+of the RTCPeerConnection API, which includes the real time media stack.
+
+## How?
+
+### General API overview
+The API has 3 main abstractions, the RTCIceTransport, RTCQuicTransport
+and RTCQuicStream:
+
+<img src="/web/updates/images/2019/01/rtcquictransport/rtcquictransport-overview.png"
+alt="RTCQuicTransport diagram showing architecture of API">
+
+### RTCIceTransport
+
+ICE is a protocol to establish peer-to-peer connections over the internet and
+is used in WebRTC today. This object provides a standalone API to establishes
+an ICE connection. It is used as the packet transport for the QUIC connection,
+and the RTCQuicTransport takes it in its constructor.
+
+### RTCQuicTransport
+
+Represents a QUIC connection. It is used to establish a QUIC connection and
+create QUIC streams. It also exposes relevant stats for the QUIC connection
+level.
+
+### RTCQuicStream
+
+Used for reading and writing data to/from the remote side. Streams transport
+data reliably and in order. Multiple streams can be created from the same
+RTCQuicTransport and once data is written to a stream it fires an “onquicstream”
+event on the remote transport. Streams offer a way to distinguish different
+data on the same QUIC connection. Common examples can be sending separate
+files across separate streams, small chunks of data across different streams,
+or different types of media across separate streams. RTCQuicStreams are
+lightweight, are multiplexed over a QUIC connection and do not cause head of
+line blocking to other RTCQuicStreams.
+
+### Setting up a connection
+The following is an example for setting up a peer-to-peer QUIC connection.
+Like RTCPeerConnection, the RTCQuicTransport API requires the use of a secure
+signaling channel to negotiate the parameters of the connection, including its
+security parameters. The RTCIceTransport negotiates it’s ICE parameters
+(ufrag and password), as well as RTCIceCandidates.
+
+__Note__ that the RTCQuicTransport connection is setup with a pre shared key
+API. We do not currently plan on keeping this API past the origin trial. It
+will be replaced by signaling remote certificate fingerprints to validate
+self-signed certificates used in the handshake, once this support has been
+added to QUIC in Chromium.
+
+<img src="/web/updates/images/2019/01/rtcquictransport/rtcquictransport-setup.png"
+alt="RTCQuicTransport diagram showing architecture of API">
+
+#### Client perspective:
+
+```
```

Comment:
To be safe, I recommend adding `js` to the end of this so the parser knows for sure it is javascript